src/platform/forms-system/src/js/fields/ArrayField.jsx

```diff
@@ -308,24 +311,26 @@ export default class ArrayField extends React.Component {
                       {notLastOrMultipleRows && (
                         <div className="row small-collapse">
                           <div className="small-6 left columns">
-                            {!isLast && (
+                            {(!isLast || showSave) && (
                               <button
                                 className="float-left"
                                 onClick={() => this.handleUpdate(index)}
-                                aria-label={`Update ${title}`}
+                                aria-label={`${updateText} ${title}`}
                               >
-                                Update
+                                {updateText}
                               </button>
                             )}
                           </div>
                           <div className="small-6 right columns">
-                            <button
-                              className="usa-button-secondary float-right"
-                              type="button"
-                              onClick={() => this.handleRemove(index)}
-                            >
-                              Remove
-                            </button>
+                            {index !== 0 && (
```

Comment:
If `index` is not zero, then the right columns will be empty, correct?

Reply:
This is set up so that the delete button is included on all, but the first card.